packages/airnode-node/src/config/types.ts

```diff
@@ -113,5 +119,6 @@ export interface Config {
   readonly nodeSettings: NodeSettings;
   readonly ois: OIS[];
   readonly triggers: Triggers;
+  readonly templates?: Template[];
```

Comment:
Let's make this required. The user can specify `[]` if he doesn't want to hardcode any template.

Reply:
Nice, I'll keep this in mind :smile: